homeassistant/components/doorbird/config_flow.py

```diff
@@ -39,7 +41,9 @@ def _check_device(device: DoorBird) -> tuple[tuple[bool, int], dict[str, Any]]:
     return device.ready(), device.info()
 
 
-async def validate_input(hass: core.HomeAssistant, data):
+async def validate_input(
+    hass: core.HomeAssistant, data: dict[str, Any]
```

Comment:
One remark tho, can't we import HomeAssistant itself here?

Reply:
We could but that would be more change in this PR. I was trying to do the bare minimum to enable strict typing to contain scope.

Reply:
Yea, I just wanted to mention it, it was already like this before so it isn't a major issue